ui/src/alerting/components/CreateCheckDropdown.tsx

```diff
@@ -0,0 +1,70 @@
+// Libraries
+import React, {FunctionComponent, MouseEvent} from 'react'
+
+// Components
+import {Dropdown, ComponentColor, IconFont} from '@influxdata/clockface'
+
+interface Props {
+  onCreateThreshold: () => void
+  onCreateDeadman: () => void
+}
+
+enum CheckType {
+  Deadman = 'deadman',
+  Threshold = 'threshold',
+}
+
+const CreateCheckDropdown: FunctionComponent<Props> = ({
+  onCreateThreshold,
+  onCreateDeadman,
+}) => {
+  const handleItemClick = (type: CheckType): void => {
+    if (type === CheckType.Threshold) {
+      onCreateThreshold()
+    }
+
+    if (type === CheckType.Deadman) {
+      onCreateDeadman()
+    }
+  }
+
+  const DropdownButton = (
+    active: boolean,
+    onClick: (e: MouseEvent<HTMLButtonElement>) => void
+  ) => (
+    <Dropdown.Button
+      icon={IconFont.Plus}
+      color={ComponentColor.Primary}
+      active={active}
+      onClick={onClick}
+      testID="create-check"
+    >
+      Create
+    </Dropdown.Button>
+  )
+
+  const DropdownMenu = (onCollapse: () => void) => (
+    <Dropdown.Menu onCollapse={onCollapse}>
+      <Dropdown.Item
+        value={CheckType.Threshold}
```

Comment:
and here (and all the other places) you can just enter the string 'threshold' when you use the type from types